### DISCUSSION TITLE
Hiring

### DISCUSSION THREAD

**Discussion by Speedy2k** [VasilKalchev/LiquidMenu#113] [General]
Hi Vasil, i would like to know if it would be possible to hire you for a very simple UI with a 20x4 lcd. I would like to have your style of coding! Once you will have made the menu system, i will be able to understand more everything of your library and update it as necessary! I don't think you will have more then 1-2 hours needed doing it!
Let me know!
Thanx!

**Comment by VasilKalchev** [comment]
Hi @Speedy2k.

Sorry I missed your post, hope you've managed fine by yourself. If you have other questions feel free to ask.
